### DISCUSSION TITLE
nitro:generate hook was removed. Which hook should replace it ?

### DISCUSSION THREAD

**Discussion by benoitdemaegdt** [nuxt/nuxt#17729] [Questions] [answered]
Hey there 👋 

Few days ago **nitro:generate** hook was removed (see this [comment](https://github.com/nuxt/framework/pull/4187#discussion_r845623039)).

However, some modules were relying on it.
Among them : 
- [partytown module](https://github.com/nuxt-community/partytown-module/blob/main/src/module.ts#L100)
- [this sitemap module](https://github.com/benoitdemaegdt/nuxt3-sitemap/blob/main/sitemap.ts#L42)

There is [this list](https://github.com/nuxt/framework/blob/main/packages/schema/src/types/hooks.ts#L55) of all hooks available at build time, but as [the documentation is still empty](https://v3.nuxtjs.org/api/advanced/hooks#nuxt-hooks-build-time), I'm not sure which one may be useful to replace nitro:generate.

Any idea ? 💡

**Comment by pi0** [comment, marked as answer]
Thanks for poiniting out to the usage. Made https://github.com/nuxt/framework/issues/4265 to add it back and avoid breaking change.
